crates/llms/tests/llms/create.rs

```diff
@@ -36,3 +42,96 @@ pub(crate) fn create_anthropic(model_id: Option<&str>) -> Result<Arc<dyn Chat>,
 
     Ok(Arc::new(model))
 }
+
+pub(crate) async fn create_local() -> Result<Arc<dyn Chat>, anyhow::Error> {
```

Comment:
```suggestion
pub(crate) async fn create_local(model_id: &str) -> Result<Arc<dyn Chat>, anyhow::Error> {
```

Reply:
As local model requires many parameters and we have hardcoded params only for one (currently support only single local model / model id) I think this is fine to have as-is, it can be added later if we will be adding more. Please let me know if I miss anything and we can specify different model id for the currently used local model params